dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/enums/DbType.java

```diff
@@ -46,7 +46,8 @@ public enum DbType {
     DAMENG(15, "dameng"),
     OCEANBASE(16, "oceanbase"),
     SSH(17, "ssh"),
-    KYUUBI(18, "kyuubi");
+    DATABEND(18, "databend"),
+    KYUUBI(19, "kyuubi");
```

Comment:
Since we've merged kyuubi's PR in #13642. It's better not to change its order.

Reply:
Thanks! Fixed it.